android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt

```diff
@@ -0,0 +1,18 @@
+package com.magicleap.magicscript.ar
+
+import com.google.ar.sceneform.animation.ModelAnimator
+import com.google.ar.sceneform.rendering.ModelRenderable
+
+class LoopedAnimator : RenderableAnimator {
+
+    override fun play(modelRenderable: ModelRenderable) {
+        val animationsNumber = modelRenderable.animationDataCount
```

Comment:
maybe `animationsCount` or `animationsSize`

Reply:
This is ARCore's property, cannot change :-)

Reply:
I mean val name

Reply:
Hah, I'm sleeping still, ok will rename it